clang/test/CodeGenSYCL/template-template-parameter.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang_cc1 -fsycl -fsycl-is-device -fsycl-int-header=%t.h %s
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -fsycl-int-header=%t.h -sycl-std=2020 %s
```

Comment:
Just a question, what is the reason to add `sycl-std` flag here?

Reply:
Without the sycl-std flag, the test generated several of these warnings : Passing of kernel functions by reference is a SYCL 2020 extension. Since these warnings are not related to the test, I added the sycl-std flag so they don't show up.

Reply:
Ah, ok. Thank you for this. Although this change doesn't relate to your patch, I'm ok with that.